lib/kitchen/directions/bake_numbered_exercise/main.rb

```diff
@@ -3,18 +3,14 @@
 module Kitchen
   module Directions
     module BakeNumberedExercise
-      def self.v1(exercise:, number:, suppress_solution: false, note_suppressed_solutions: false)
-        V1.new.bake(exercise: exercise, number: number, suppress_solution: suppress_solution,
+      def self.v1(exercise:, number:, suppress_solution_if: false, note_suppressed_solutions: false)
+        V1.new.bake(exercise: exercise, number: number, suppress_solution_if: suppress_solution_if,
                     note_suppressed_solutions: note_suppressed_solutions)
       end
 
       def self.bake_solution_v1(exercise:, number:, divider: '. ')
         V1.new.bake_solution(exercise: exercise, number: number, divider: divider)
       end
-
-      def self.bake_even_solution_v1(exercise:, divider: '. ')
```

Comment:
I have also removed `bake_even_solution_v1` method, because it is not needed anymore after your changes :slightly_smiling_face: :